test/src/api/commands/testUploadFile.js

```diff
@@ -0,0 +1,50 @@
+const assert = require('assert');
```

Comment:
Some kind of stub or interceptor is needed for testing the selenium FileDetector api is called. If that's not feasible, we can then instead add a protocol test.

Reply:
sure, make sense